model.py

```diff
@@ -1050,6 +1051,9 @@ class Identifier(Base):
     # One Identifier may have many associated CoverageRecords.
     coverage_records = relationship("CoverageRecord", backref="identifier")
 
+    # One Identifier may be registered in many collections.
+    collections = relationship("CollectionAsset", backref="identifier")
+
```

Comment:
This will give you a list of CollectionAsset objects. Either this needs to be called collection_assets (which I don't recommend because CollectionAsset looks like a pure join table) or you need to make this a many-to-many relationship a la http://docs.sqlalchemy.org/en/rel_0_9/orm/basic_relationships.html#many-to-many, so that identifier.collections actually gives you a list of Collection objects.